packages/video_player/lib/video_player.dart

```diff
@@ -148,14 +150,16 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
   /// package and null otherwise.
   VideoPlayerController.asset(this.dataSource, {this.package})
       : dataSourceType = DataSourceType.asset,
+        formatHint = null,
         super(VideoPlayerValue(duration: null));
 
   /// Constructs a [VideoPlayerController] playing a video from obtained from
   /// the network.
   ///
   /// The URI for the video is given by the [dataSource] argument and must not be
-  /// null.
-  VideoPlayerController.network(this.dataSource)
+  /// null. The [formatHint] option allows the caller to specify the video
```

Comment:
nits:
Start `The [formatHint]...` on a new line.
Also indicate it is Android only.